docs/README.md

```diff
@@ -17,19 +17,16 @@
 
 # Building Documentation
 
-A basic python environment with packages listed in `./requirement.txt` is
-enough to build the docs.
+Additional packages required for building the documentation are defined in `./conda_docs.yml`.
 
 ## Get additional dependency
```

Comment:
should this be `Get Additional Dependencies` to align with other headers?

Reply:
I went with `Install Additional Dependencies`